addons/web/static/src/core/dialog/dialog.xml

```diff
@@ -19,7 +19,7 @@
                         </header>
                         <!-- FIXME: WOWL there is a bug on t-portal on owl, in which t-portal don't work on multinode.
                         To avoid this we place the footer before the body -->
-                        <footer t-if="props.footer" class="modal-footer justify-content-around justify-content-sm-start flex-wrap gap-1 w-100" style="order:2">
+                        <footer t-if="props.footer" class="modal-footer flex-column flex-sm-row justify-content-sm-start flex-wrap gap-1 w-100" style="order:2">
```

Comment:
Out of scope, read my comment in `addons/web/static/src/core/dialog/dialog.scss` 🙏🏼